src/OrchardCore.Modules/OrchardCore.Settings/ResourceConfigureOptions.cs

```diff
@@ -0,0 +1,30 @@
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Options;
+using OrchardCore.ResourceManagement;
+
+namespace OrchardCore.Settings;
+
+public class ResourceConfigureOptions : IConfigureOptions<ResourceOptions>
+{
+    private readonly ISiteService _siteService;
+    private readonly IHttpContextAccessor _httpContextAccessor;
```

Comment:
```suggestion
    private readonly HttpContext _httpContext;
```

Reply:
What is the added value from doing this, if any?

Reply:
Time to check the entire source code ;)